<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationDialog</class>
 <widget class="QDialog" name="RegistrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Registration Dialog Style */
QDialog {
    background-color: #f5f5f5; /* Светло-серый фон для диалогового окна */
    border: 1px solid #dcdcdc; /* Граница окна */
}

/* Line Edits */
QLineEdit {
    border: 2px solid #dcdcdc; /* Граница для полей ввода */
    border-radius: 5px; /* Скругленные углы */
    padding: 5px; /* Отступ внутри поля ввода */
    font-size: 14px; /* Размер шрифта */
    background-color: #ffffff; /* Белый фон */
}

/* Combo Boxes */
QComboBox {
    border: 2px solid #dcdcdc; /* Граница для комбо-бокса */
    border-radius: 5px; /* Скругленные углы */
    padding: 5px; /* Отступ внутри комбо-бокса */
    font-size: 14px; /* Размер шрифта */
    background-color: #ffffff; /* Белый фон */
}

/* Buttons */
QPushButton {
    background-color: #0078d7; /* Синий фон кнопки */
    color: #ffffff; /* Белый цвет текста */
    border: none; /* Убираем границу */
    border-radius: 5px; /* Скругленные углы */
    padding: 10px; /* Отступ внутри кнопки */
    font-size: 14px; /* Размер шрифта */
}

QPushButton:hover {
    background-color: #005bb5; /* Темно-синий фон при наведении */
}

QPushButton:pressed {
    background-color: #004494; /* Еще более темный фон при нажатии */
}

/* Labels */
QLabel {
    font-size: 14px; /* Размер шрифта */
    color: #333333; /* Темно-серый цвет текста */
}

/* Message Boxes */
QMessageBox {
    background-color: #ffffff; /* Белый фон для окон сообщений */
    border: 1px solid #dcdcdc; /* Граница окна сообщения */
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="familyLineEdit">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="patronymicLineEdit">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="loginLineEdit">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Роль пользователя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="roleComboBox">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Преподаватель</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Учебно-вспомогательный персонал</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Степень/должность</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="degreeComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="registerButton">
     <property name="text">
      <string>Зарегистрироваться</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
